app/src/components/MessageList.tsx

```diff
@@ -221,6 +263,9 @@ class MessageList extends Component<Props, State> {
             <div className={classes.scrollContainer}>
                 <Box className={classes.tableHeaderRow}>
                     <Box className={classNames(classes.tableColumn, classes.overrideTimestampColumn)}>
+                        <CustomTooltip title={'Click the Tilde ` to manually center to selected message'}>
+                            <InfoIcon />
+                        </CustomTooltip>
```

Comment:
can this custom tooltip have a space after it so that it's a little separated from "Timestamp" label

Reply:
👍